src/devices/bus/neogeo/prot_pvc.cpp

```diff
@@ -135,7 +136,7 @@ void pvc_prot_device::mslug5_decrypt_68k(uint8_t* rom, uint32_t size)
 }
 
 
-void pvc_prot_device::svc_px_decrypt(uint8_t* rom, uint32_t size)
+void pvc_prot_device::svc_decrypt_68k(uint8_t* rom, uint32_t size)
```

Comment:
NeoGeo refers to the two 16-bit code address spaces as P/P0/ROM (read-only, 16-bit only) and P1/PORT (read/write, selectable 8-bit lanes) - the original name of this function makes sense.